skycultures/Book_of_Fixed_Stars_English/info.ini

```diff
@@ -0,0 +1,7 @@
+﻿[info]
+name = Book of Fixed Stars
+Author : Khalid AlAjaji

```

Comment:
author=Khalid AlAjaji
And again, newline at end please

Reply:
done